dev/devicelab/lib/tasks/track_widget_creation_enabled_task.dart

```diff
@@ -46,10 +46,6 @@ class TrackWidgetCreationEnabledTask {
           deviceIdOverride,
           path.join('lib/track_widget_creation.dart'),
         ]),
-        environment: <String, String>{
-          'FLUTTER_WEB': 'true',
-          'FLUTTER_MACOS': 'true'
```

Comment:
I also see one in hot_mode_dev_cycle_macos_target__benchmark

Reply:
Oh and I see other `FLUTTER_WEB : true` in other devicelab tasks, can you do a sweep?

Reply:
Removed FLUTTER_WEB, FLUTTER_MACOS that are set by config


FLUTTER_EXPERIMENTAL_BUILD is dead code too